ui/src/components/LoginForm/SelectProposal.jsx

```diff
@@ -77,7 +77,7 @@ function SelectProposal() {
           onChange={(evt) => setFilter(evt.target.value)}
         />
 
-        <Tabs id="scheduled-tab" defaultActiveKey="scheduled">
```

Comment:
Might be worth changing the ID as well? To be double checked, but I think it's just for accessibility.